src/services/Gists.js

```diff
@@ -63,22 +62,7 @@ function updateGistWithImportUrl(github, gistData) {
 
   return gist.update({
     description: `${gistData.description} Click to import: ${uri.href}`,
-  }).then(
-    (response) => response.data,
-    notifyAndRejectApiError
-  );
-}
-
-function notifyAndRejectApiError(error) {
-  if (error.response) {
-    Bugsnag.notifyException(
-      error,
-      'GistError',
-      {failedRequest: pick(error, ['request', 'response', 'status'])}
-    );
-  }
-
-  return Promise.reject(error);
```

Comment:
We don’t really need this anymore, because the newer version of the GitHub client now gives us useful error information.